scripts/commands/addcurrency.lua

```diff
@@ -9,33 +9,125 @@ cmdprops =
     parameters = "sis"
 };
 
-function onTrigger(player,currency,amount,target)
-    local targ = nil
+function error(player, msg)
+    player:PrintToPlayer(msg);
+    player:PrintToPlayer("@addcurrency <currency type> <amount> {player}");
+end;
 
-    if (currency == nil or amount == nil) then
-        player:PrintToPlayer( "You must enter a valid currency and amount." );
-        player:PrintToPlayer( "@addcurrency <currency type> <amount> <target player>" );
-        return
-    end
+function onTrigger(player,currency,amount,target)
+    local currencies = {
+        sandoria_cp=true,
```

Comment:
Kind of how its intended though, because if a script uses an invalid currency it should throw the error.\* I'll wait and see if other dev's want the sql error axe'd this way. 

\**I admit our sql error handling needs work though - we're using the generic "it didn't work" which is useless 90% the time unless you are the only person on the test server.*